woocommerce/woocommerce-gutenberg-products-block
# Products by Rating: Add display options

Add support for the following display options:
- Product count
- 'Apply filters' button
- Typography - Size
- Color 

Fixes #7171 

### Screenshots
![190835898-f62d5dff-8921-44ea-acf2-d5f690544ef7](https://user-images.githubusercontent.com/905781/194179286-b6e8fbf6-37a8-42fc-95bb-e122dde2b694.jpg)

### Testing
1. Activate a block theme & setup WooCommerce with products that have some ratings
2. Create a classic shop page and a separate page using the All Products block
3. Add the `Filter Products by Rating` block and `Active Filters` block on both pages
4. Verify that all Display Options display and work correctly both in the editor and the page

### WooCommerce Visibility


### Changelog

> Products by Rating: Add support for Display Options
